src/components/DepositWidget/index.tsx

```diff
@@ -346,14 +346,14 @@ const DepositWidget: React.FC = () => {
     <section>
       <BalancesWidget>
         <BalanceTools>
```

Comment:
Actually @anxolin we could also use a single 'not-implemented' class for the whole BalanceTools element. So that it doesn't output at all.

Reply:
Yes, I was going to do it like that, but since they have independent issues. I thought it was better to do it one by one, so we can implement the issues independently with less conflicts